src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/models/Policy.kt

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexstatemanagement.models
+
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.elasticapi.instant
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.elasticapi.optionalTimeField
+import org.elasticsearch.common.lucene.uid.Versions
+import org.elasticsearch.common.xcontent.ToXContent
+import org.elasticsearch.common.xcontent.ToXContentObject
+import org.elasticsearch.common.xcontent.XContentBuilder
+import org.elasticsearch.common.xcontent.XContentParser
+import org.elasticsearch.common.xcontent.XContentParser.Token
+import org.elasticsearch.common.xcontent.XContentParserUtils.ensureExpectedToken
+import java.io.IOException
+import java.time.Instant
+
+data class Policy(
+    val id: String = NO_ID,
+    val version: Long = NO_VERSION,
+    val name: String,
+    val schemaVersion: Long,
+    val lastUpdatedTime: Instant,
+    // TODO: Implement DefaultNotification(destination, message)
+    val defaultNotification: Map<String, Any>?,
+    val defaultState: String,
+    val states: List<Map<String, Any>> // TODO: Implement List<State>
+) : ToXContentObject {
+
+    fun toXContent(builder: XContentBuilder): XContentBuilder {
+        return toXContent(builder, ToXContent.EMPTY_PARAMS)
+    }
+
+    override fun toXContent(builder: XContentBuilder, params: ToXContent.Params): XContentBuilder {
+        builder
+            .startObject()
+                .startObject(POLICY_TYPE)
+                    .field(NAME_FIELD, name)
+                    .optionalTimeField(LAST_UPDATED_TIME_FIELD, lastUpdatedTime)
+                    .field(SCHEMA_VERSION_FIELD, schemaVersion)
+                    .field(DEFAULT_NOTIFICATION_FIELD, defaultNotification)
+                    .field(DEFAULT_STATE_FIELD, defaultState)
+                    .field(STATES_FIELD, states.toTypedArray())
+                .endObject()
+            .endObject()
+        return builder
+    }
+
+    companion object {
+        const val POLICY_TYPE = "policy"
+        const val NAME_FIELD = "name"
+        const val NO_ID = ""
+        const val NO_VERSION = Versions.NOT_FOUND
```

Comment:
Is there a reason why ```NO_VERSION``` here is ```Versions.NOT_FOUND``` but ```1L``` for ```ManagedIndex```?

Reply:
More explicit, will be changing ManagedIndex